### PR TITLE
Docs: Update migrating-to-2.0.0.md

### PR DESCRIPTION
**Pull request by mysticatea**
I added [New Rules in `"eslint:recommended"`] section.

### CODE REVIEW

File: docs/user-guide/migrating-to-2.0.0.md [hunk 1]
```diff
@@ -183,6 +183,48 @@ Additionally, if you are using `context.ecmaFeatures` inside of your rules, then
...
+* [no-const-assign](http://eslint.org/docs/rules/no-const-assign)
+* [no-dupe-class-members](http://eslint.org/docs/rules/no-dupe-class-members)
+* [no-empty-pattern](http://eslint.org/docs/rules/no-empty-pattern)
+* [no-new-symbol](http://eslint.org/docs/rules/no-new-symbol)
+* [no-self-assign](http://eslint.org/docs/rules/no-self-assign)
+* [no-this-before-super](http://eslint.org/docs/rules/no-this-before-super)
+* [no-unexpected-multiline](http://eslint.org/docs/rules/no-unexpected-multiline)
+* [no-unused-labels](http://eslint.org/docs/rules/no-unused-labels)
+
+**To address:** If you don't want to be notified by those rules, you can disable those rules simply.
```
Comment (hzoo) [inline docs/user-guide/migrating-to-2.0.0.md]: Maybe `you can simply disable those rules`?

Comment (mysticatea) [inline docs/user-guide/migrating-to-2.0.0.md]: Thank you for correction!

File: docs/user-guide/migrating-to-2.0.0.md [hunk 2]
```diff
@@ -183,6 +183,48 @@ Additionally, if you are using `context.ecmaFeatures` inside of your rules, then
...
 
+## New Rules in `"eslint:recommended"`
+
+```json
+{
+    "extends": "eslint:recommended"
+}
+```
+
+In 2.0.0, `"eslint:recommended"` includes the following 11 rules anew.
```
Comment (ilyavolodin) [inline docs/user-guide/migrating-to-2.0.0.md]: In 2.0.0, the following 11 rules were added to `"eslint:recommended"`.

Comment (mysticatea) [inline docs/user-guide/migrating-to-2.0.0.md]: Thank you for correction!

**Pull request merged**